dietpi/dietpi-software

```diff
@@ -12091,6 +12098,73 @@ If no WireGuard (auto)start is included, but you require it, please do the follo
 
 			unset -v choice_required domain port invalid_text server_ip
 		fi
+
+		if To_Install 213 # soju
+		then
+			local ver=0.8.0
+			aDEPS=('gcc' 'make' 'pkg-config' 'scdoc' 'libpam-dev' 'libsqlite3-dev')
+			Download_Install "https://git.sr.ht/~emersion/soju/archive/v${ver}.tar.gz"
+			G_EXEC cd "soju-v${ver}"
+
+			# Build
+			G_EXEC_OUTPUT=1 G_EXEC make all "-j$(nproc)" \
+				config_path=/mnt/dietpi_userdata/soju/config
+				GOFLAGS=-tags=pam,libsqlite3 \
+
+			# Install
+			local commands=('soju' 'sojuctl' 'sojudb')
+			local manpages=('doc/soju.1' 'doc/sojuctl.1')
+			G_EXEC mkdir -p /usr/local/bin
+			G_EXEC mkdir -p /usr/local/share/man/man1
```

Comment:
Is one of the commands commonly used for CLI admin operations as well? Usually, we do not install stray files and executables to `/usr/local`. In most cases, they are used to start the service only, where the location does not matter. In case of a CLI, usually, one must run it as service user, to not mess with config/data permissions. And to make this easier, we do then create a shell alias which implies `sudo -u <service_user> /path/to/exec`, so the CLI is automatically called correctly, and again, the path does not matter. So executables and everything which is variable data/configs, is then stored to `/opt/<software_name>` instead.

Reply:
I left it like that for now. `sojuctl` is a common CLI tool, and the installed commands and manpages are well defined. We can move then any time later, add an alias etc, if it really turns out to cause issues when using it with any other user than `soju`.